### ISSUE TITLE
String labels in scikit-learn KerasClassifier

### ISSUE THREAD

**Issue by reiinakano** [fchollet/keras#5055]
Currently, scikit-learn accepts strings as labels in its classification algorithms. I believe a similar functionality could be added to KerasClassifier to make it a bit closer in being a drop-in replacement for an sklearn estimator. I've done something similar in one of my use cases and could submit a PR, unless of course I've missed something obvious as to why this couldn't be implemented in KerasClassifier.
